Dockerfile

```diff
@@ -46,15 +46,10 @@ CMD yarn start
 FROM base AS ci
 
 # CI pipeline utilities
-# python3-* is needed for pre-commit
 RUN apt-get install -y --quiet --no-install-recommends \
-    python3 \
     make \
     bash \
     sed \
-    python3-distutils \
-    python3-pkg-resources \
     gcc \
-    python3-dev \
     libc-dev && \
     apt-get autoremove -y
```

Comment:
This all can fit nicely into the base image, having a separate CI image doesn't make much sense anymore

Reply:
@alvicsam recommended to leave as is, he will clean this up in https://github.com/paritytech/ci_cd/issues/690 💪